clouddrift/analysis.py

```diff
@@ -40,15 +44,18 @@ def apply_ragged(
     ----------
     func : callable
         Function to apply to each row of each ragged array in ``arrays``.
-    arrays : list[np.ndarray] or np.ndarray
+    arrays : list[np.ndarray] or np.ndarray or xr.DataArray
         An array or a list of arrays to apply ``func`` to.
-    rowsize : list
+    rowsize : list[int] or np.ndarray[int] or xr.DataArray[int]
         List of integers specifying the number of data points in each row.
     *args : tuple
         Additional arguments to pass to ``func``.
     rows : int or Iterable[int], optional
         The row(s) of the ragged array to apply ``func`` to. If ``rows`` is
         ``None`` (default), then ``func`` will be applied to all rows.
+    axis : int, optional
+        The axis along which to unpack the input ragged array(s) and concatenate
```

Comment:
The axis of the input ragged arrays that is ragged?

Reply:
Can you expand? I don't understand. You can also use the suggest feature to apply a specific edit.

Reply:
I suggest to rephrase the definition of this argument to something like: "The axis of the input array that is ragged". The user does not necessarily know what unpack means in this context or why you mention concatenate etc.